### ISSUE TITLE
Question about the shapeworld dataset

### ISSUE THREAD

**Issue by hlml** [jayelm/lsl#10]
Hi Jesse,

Quick question about the splits in shapeworld: what's the difference between the `val` and `val_same` (and `test` vs `test_same`) data splits?

Thanks!
Hattie

**Comment by jayelm** [comment]
`val_same` is a validation set with new tasks, but the concepts (e.g. `red square above blue triangle`) overlap with those in the training set. `val` has new tasks and also completely new concepts (same for test). Hope that clarifies?

**Comment by hlml** [comment]
Ah ok perfect, thanks a lot!
